doc/nrf/libraries/others/identity_key.rst

```diff
@@ -0,0 +1,41 @@
+.. _lib_identity_key:
+
+Identity key
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The identity key library manages an asymmetric key used for identity services on devices with the Arm CryptoCell and KMU peripherals.
+
+Functionality
+*************
+
+This library manages the identity keys, which are asymmetric keys intended to provide identity to specific devices.
+The identity key is designed to be unique and is provisioned either device-generated or otherwise in a secure manner during production.
+The identity key is encrypted and stored in the Key Management Unit (KMU), in two reserved KMU slots.
+The HUK used to encrypt the identity key is the Master Key Encryption Key (MKEK), which is a key reserved for encrypting key-wallet keys.
+The identity key is only available for use from a secure image, like the TF-M image, a bootloader, or an image used during production of the device.
+
+.. caution::
+   The identity key must not be shared. Leaking this leaks the identity of the device.
+
+.. caution::
+   The identity key stored in KMU will be erased when ERASEALL is called.
```

Comment:
```suggestion
   The identity key is stored in the KMU and will therefore be erased by ERASEALL.
```

Using a modifier on "identity key" sounds like there are multiple identity keys and this just applies to the one stored in the KMU.

called implies a function call, but ERASEALL can come from the debugger or from MMIO.

Reply:
done